pkg/dispatchcli/cmd/update_baseimage.go

```diff
@@ -66,37 +49,10 @@ func NewCmdUpdateBaseImage(out io.Writer, errOut io.Writer) *cobra.Command {
 		},
 	}
 
-	cmd.Flags().StringVar(&imageURL, "image-url", "", "The url for the container image.")
-	cmd.Flags().StringVar(&language, "language", "", "Specify the runtime language for the image")
 	return cmd
 }
 
 func updateBaseImage(out io.Writer, errOut io.Writer, cmd *cobra.Command, args []string) error {
-	baseImageName := args[0]
-
-	params := base_image.NewGetBaseImageByNameParams()
-	params.BaseImageName = baseImageName
-
-	ok, err := imageManagerClient().BaseImage.GetBaseImageByName(params, GetAuthInfoWriter())
-	if err != nil {
-		return errors.Wrapf(err, "Failed retrieving base-image named : %s", baseImageName)
-	}
-
-	baseImage := *ok.Payload
-	baseImage.Name = &baseImageName
-	if cmd.Flags().Changed("image-url") {
-		baseImage.DockerURL = &imageURL
-	}
-
-	if cmd.Flags().Changed("language") {
-		baseImage.Language = swag.String(language)
-	}
-
-	err = CallUpdateBaseImage(&baseImage)
-	if err != nil {
-		return err
-	}
-
-	fmt.Fprintf(out, "Updated base image: %s\n", baseImageName)
+	//TODO: Implement file writer update mechanism
```

Comment:
Should we just remove all of the update_*.go files?  Seems none of them currently work.  We are waiting for the editor support, correct?

Reply:
That's right. I can remove all of them and just move all the Call*Update methods into the update.go file.